test/functional/signature/version-sig-check.bats

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bats
+
+# Author: Karthik Prabhu
+# Email: karthik.prabhu.vinod@intel.com
+
+load "../testlib"
+
+test_setup() {
+
+	create_test_environment "$TEST_NAME"
+	create_bundle -L -n test-bundle1 -f /file_1 "$TEST_NAME"
+	create_version -p "$TEST_NAME" 20 10
+	update_bundle "$TEST_NAME" test-bundle1 --add file_3
+
+}
+
+@test "SIG019: Verify signature for absolute latest version with swupd check-update" {
+
+	# absolute latest version get checked during check-update
+	sudo sh -c "mv $WEBDIR/version/latest_version.sig $WEBDIR/version/latest_version_bad_name.sig"
+	run sudo sh -c "$SWUPD check-update $SWUPD_OPTS"
+        assert_status_is "SWUPD_SIGNATURE_VERIFICATION_FAILED"
+
```

Comment:
I would like to include checking for the output here too, I like checking the output to see if all messages make sense in case of an error.

Reply:
Sounds good. Will have to do a regex though as there are file name involved